package.json

```diff
@@ -55,11 +55,13 @@
   ],
   "dependencies": {
     "@babel/polyfill": "^7.8.3",
-    "@datapunt/asc-assets": "^0.18.1",
-    "@datapunt/asc-core": "^0.18.1",
-    "@datapunt/asc-ui": "^0.18.2",
+    "@datapunt/amsterdam-react-maps": "git+https://github.com/Amsterdam/amsterdam-react-maps.git#feature/package-export",
```

Comment:
Can't we use a specific version reference?

Reply:
there are not yet versions defined in this component

Reply:
This notation won't work as Jenkins would need Github authentication tokens. We cannot merge this PR otherwise.

Reply:
It works, It;s already implemented in another project